pymc/distributions/continuous.py

```diff
@@ -3121,7 +3121,11 @@ def triangular_default_transform(op, rv):
 
 class Gumbel(Continuous):
     r"""
-    Univariate Gumbel log-likelihood.
+    Univariate Gumbel right Skew log-likelihood.
+
+    This parameterization will provide the extreme maximum value.
+    Those looking to find the extreme minimum provided by the left skewed Gumbel should
+    invert the sign to all X and Y values (positive to negative and negative to positive).
```

Comment:
The original PR https://github.com/pymc-devs/pymc/pull/6478 referred to this [SciPy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.gumbel_r.html) reference. There, the `y` is equivalent of the `z` used in [our documentation](https://www.pymc.io/projects/docs/en/latest/api/distributions/generated/pymc.Gumbel.html).

However, I believe it would be better to modify the text to point changes in the signals of all `x` and `mu`, (which is equivalent to changing the signal of  all `x` and `z`) because we have control over the `mu` as an arguments of the function, but not `z` (I believe). 

I have tested this and our implementation matches scipy.stats.gumbel_r (for Max) and scipy.stats.gumbel_l (for Min, with negative x and mu passed to our function).

<img width="617" alt="image" src="https://github.com/pymc-devs/pymc/assets/22087841/0671132c-1311-4e61-a98c-60413203915a">

I have corrected the text to be "Those looking to find the extreme minimum provided by the left-skewed Gumbel should invert the sign to all x and mu values."

Let me know if this make sense or we prefer to refer `z` instead of mu since it is defined in our documentation below the pdf formula.